Source/WebCore/dom/RenderedDocumentMarker.h

```diff
@@ -68,9 +70,32 @@ class RenderedDocumentMarker : public DocumentMarker {
 
     bool isValid() const { return m_isValid; }
 
+    float opacity() const { return m_opacity; }
+    void setOpacity(float opacity) { m_opacity = opacity; }
+
+    bool isActive() const { return m_isActive; }
```

Comment:
I'm not sure I like 'active' for this state, especially if it's more of a transitional state that will only be around when fading. I think that 'fading/isFading' or 'deactivating/isDeactivating' is a better term for an actively transitionary state, rather than 'inactive' which makes me thing of a marker that is still around, but not being painted for some reason.

Reply:
I will rename to `isBeingDismissed`, since that aligns with the method name `dismissMarkers`.